.ci.yaml

```diff
@@ -55,6 +55,7 @@ targets:
     recipe: engine_v2/engine_v2
     properties:
       config_name: linux_android_emulator
+      test_timeout_secs: "5400" # 90 minutes
```

Comment:
The test will timeout after at most 60 mins, as the subbuild's execution timeout is 60 mins. And the subbuild's timeout number is consistent with orchestrator's. Where do you see the 90 mins?

Reply:
On line 63, `timeout: 90`.

Reply:
If this doesn't change the subbuilds execution time, then what does? I don't see any other way to do it.

Reply:
My bad using an outdate local config repo (which missed the recent change: https://github.com/flutter/engine/pull/50954)

Subbuild's execution_timeout is not controllable from .ci.yaml side as of now. All subbuilds of different targets are sharing the same Drone builder. We change it to 90, but it will affect all drone builds. This should work for other targets (whose orchestrator is with 60), the subbuild will be killed anyway after 60 mins, but it may cause confusion where subbuild's timeout is bigger than the orchestrator's.

As in this PR, we are reducing test step timeout from 30 to 15mins, and the retry from 3 to 2, do we still need the 90 timeout for subbuild?

Reply:
Is there an issue filed for allowing controlling subbuild execution timeout? That seems like a pretty basic feature, and the existing ci properties lead one to believe that it is controllable.

Reply:
I removed the `test_timeout_secs` field.

Reply:
https://github.com/flutter/flutter/issues/144243 to track.